.travis.yml

```diff
@@ -2,7 +2,6 @@ language: rust
 rust:
   - stable
   - nightly
-cache: cargo
```

Comment:
Why remove the cargo cache?

Reply:
1:07 with, 37min without

Reply:
That is unexpected and weird. I don't see any reason why it would be faster without.

We can certainly experiment with that but please let's keep it out of this PR.